addons/godot_rl_agents/sensors/sensors_3d/GridSensor3D.gd

```diff
@@ -113,7 +136,7 @@ func _create_cell(i:int, j:int, position: Vector3):
 	
 	cell.collision_layer = 0
 	cell.collision_mask = detection_mask
-	cell.monitorable = false
+	cell.monitorable = true
```

Comment:
This allows the grid sensor to detect static objects (e.g. walls).
Didn't test the effect on performance. If there is some, a flag to turn on/off could be added later.